### DISCUSSION TITLE
HPE_UNEXPECTED_CONTENT_LENGTH :  Content-Length can't be present with Transfer-Encoding

### DISCUSSION THREAD

**Discussion by sanket-farande** [chimurai/http-proxy-middleware#974] [Q&A]
How to resolve this issue ? 

- I am using createProxyMiddleware({target: this.url, changeOrigin: true, ws: true, timeout: 6000000, secure: false});
- Somewhere compression is happening inside middleware which is adding **Transfer-Encoding** header I guess. I am not explicitly adding any compression middleware on express app.
- I tried removing header from onProxyRes but it didn't work.
onProxyRes : responseInterceptor( res.removeHeader('Content-Length'); OR delete proxyRes.headers['transfer-encoding']; OR res.removeHeader('Transfer-Encoding'); ) 
- My target is getting partially loaded but with broken css and it keeps on retrying with same error for each request.
- Example of request and response headers without proxying
![image](https://github.com/chimurai/http-proxy-middleware/assets/46294232/588c89b3-6b59-4da2-bf7d-6d83087a8211)

**Comment by chimurai** [comment]
fixed in: https://github.com/chimurai/http-proxy-middleware/pull/1249
